sdk/communication/azure-communication-administration/CHANGELOG.md

```diff
@@ -1,4 +1,32 @@
 # Release History
+## 1.0.0b3 (Unreleased) 
+
+**Breaking Changes**
+
+##### `PhoneNumberSearch` renamed to `PhoneNumberReservation`
+
+##### `PhoneNumberReservation`
+- `search_id` has been renamed to `reservation_id`
+
+##### `PhoneNumberAdministrationClient`
+- `get_search_by_id` has been renamed to `get_reservation_by_id`
+- `create_search` has been renamed to `begin_reserve_phone_numbers`
+-  `begin_reserve_phone_numbers` now takes either `options`, or `continuation_token` keywords as input
+-  `begin_reserve_phone_numbers` now returns LROPoller[PhoneNumberReservation]
```

Comment:
maybe `LROPoller[PhoneNumberReservation]`?

Reply:
Thanks. Back-quote has been added to missed places.